pkg/provider/types.go

```diff
@@ -752,6 +752,9 @@ type PresetProvider interface {
 	SetCloudCredentials(userInfo *UserInfo, presetName string, cloud kubermaticv1.CloudSpec, dc *kubermaticv1.Datacenter) (*kubermaticv1.CloudSpec, error)
 }
 
+type VersionProvider interface {
+}
+
```

Comment:
A quick ctrl-f in Firefox did not show any spots where this is being used. Am I wrong? :)

Reply:
No, you are right, probably it is garbage left from the previous idea.